pkgs/applications/networking/instant-messengers/qq/default.nix

```diff
@@ -0,0 +1,94 @@
+{ alsa-lib
+, cups
+, dpkg
+, fetchurl
+, gjs
+, glib
+, gtk3
+, gtk4
+, lib
+, libdrm
+, libgcrypt
+, libkrb5
+, mesa # for libgbm
+, nss
+, xorg
+, systemd
+, stdenv
+, at-spi2-core
+, autoPatchelfHook
+, wrapGAppsHook
+, copyDesktopItems
+, makeDesktopItem
+}:
+
+let
+  mirror = "https://dldir1.qq.com/qqfile/qq/QQNT/50eed662";
+in stdenv.mkDerivation rec {
+
+  pname = "qq";
+  version = "2.0.3-543";
+
+  src = fetchurl {
+    url = "${mirror}/QQ-v${version}_x64.deb";
+    sha256 = "sha256-O8zaVHt/oXserPVHe/r6pAFpWFeLDVsiaazgaX7kxu8=";
+  };
+
+  unpackCmd = "mkdir root; ${dpkg}/bin/dpkg-deb -x $curSrc root";
```

Comment:
have you ever read about multiline strings?

Reply:
to be fair, the command isn't that long.